src/utils.js

```diff
@@ -219,3 +218,10 @@ export function update(
 
   return set(state, path, next)
 }
+
+/**
+ * A couple of methods use identity functions. This avoids duplication.
+ */
+export function identity<T>(n: T): T {
```

Comment:
It's a generic. This is saying "This function will return the same type as it was given"